driver/desktop/key.go

```diff
@@ -14,3 +16,31 @@ const (
 	// KeyMenu represents the left or right menu / application key
 	KeyMenu fyne.KeyName = "Menu"
 )
+
+// Modifier captures any key modifiers (shift etc) pressed during this key event
+type Modifier int
+
+const (
+	// ShiftModifier represents a shift key being held
+	ShiftModifier Modifier = 1 << iota
+	// ControlModifier represents the ctrl key being held
+	ControlModifier
+	// AltModifier represents either alt keys being held
+	AltModifier
+	// SuperModifier represents either super keys being held
+	SuperModifier
+)
+
+func modifierToString(mods Modifier) string {
```

Comment:
This does not handle modifier combinations (ctrl+shift).
Maybe it should be in the desktop/shortcut.go file if it's just used for the name?